ic-helper.ps1

```diff
@@ -72,17 +72,15 @@ function Find-GitRepositories {
 $foundRepository = $false
 
 try {
-    if (-not (Test-Path $path)) {
-        throw "The specified path '$path' does not exist."
-    }
-    $subdirs = Get-ChildItem -Path $path -Directory -Recurse
+    $subdirs = Get-ChildItem -Path $path -Directory -Recurse -ErrorAction Stop
+} catch [System.Management.Automation.ItemNotFoundException] {
+    # Catch if the specified path exists, and throw an error if it doesn't
+    throw "The specified path '$path' does not exist."
+} catch {
```

Comment:
Extra `catch` block is not needed

Reply:
I tried to run with only one catch block and the error is not cathched, I do not know why...

Reply:
Okay, then let's stick to `catch` block without specifying the error type. I thought it would work.

Reply:
I have changed this part to use only one catch block and without specifying the error type